ext/standard/tests/network/dns_get_record_caa.phpt

```diff
@@ -1,38 +0,0 @@
---TEST--
-dns_get_record() CAA tests
```

Comment:
Is this one actually unstable? Unlike the others, this one works for me.

Reply:
For me it is failing currently and it seems to be a bit unstable also taking the [QA reports](http://qa.php.net/reports/viewreports.php?version=7.2.10&test=%2Fext%2Fstandard%2Ftests%2Fnetwork%2Fdns_get_record_caa.phpt) into account.